src/main/java/org/benetech/mathshare/Application.java

```diff
@@ -6,6 +6,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@SuppressWarnings("PMD.UseUtilityClass")
```

Comment:
Is there any sensible possibility to bypass this warning?

Reply:
Unfortunately, the PMD, just like Checkstyle, treats our spring main class as a utility class - it has only public, static methods. Those classes should not expose public constructor, as they shouldn't be instantiated. In our case, we need this public constructor.